### ISSUE TITLE
feature request: module federation

### ISSUE THREAD

**Issue by ngjuping** [logue/vite-vue2-vuetify-ts-starter#8]
This template worked fine for our corporation for months, a huge thanks from me and the team! However, we have now come to a stage where we need to develop functionalities independently. I have heard and dug into this term "module federation" as a paradigm for frontend micro-service, and I was wondering if there's any possibility that an integration of the "vite-plugin-federation" library into this framework is possible. 

TLDR: Any suggestion or possible upgrades in the future such that frontend microservice paradigm is possible?

Best regards,
JU PING

**Comment by logue** [comment]
I've only skimmed [vite-plugin-federation](https://github.com/originjs/vite-plugin-federation), but I think it probably won't work properly unless you unset `manualChunks` in `vite.config.ts`.

https://github.com/logue/vite-vue2-vuetify-ts-starter/blob/7dc52c7ec26eadbafa15a0af395c0b02a2d0b408/vite.config.ts#L111-L131
